### PR TITLE
Annotations query for grinder

### PR DESCRIPTION
**Pull request by goru97**
Added annotations query endpoint to grinder.

### CODE REVIEW

File: contrib/grinder/scripts/query.py [hunk 1]
```diff
@@ -96,13 +96,30 @@ def generate(self, time, logger):
...
+
+class AnnotationsQuery(AbstractQuery):
+  query_interval_name = 'annotations_queries_per_interval'
+  query_name = "AnnotationsQuery"
+  test_number = 5
+
+
+  def generate(self, time, logger):
+    tenant_id = random.randint(0, default_config['num_tenants'])
+    to = time
```
Comment (GeorgeJahad) [inline contrib/grinder/scripts/query.py]: I think Tilo has changed annotations ingest to use it's own num_tenants; if so you'll want to use that here as well.

File: contrib/grinder/scripts/tests.py [hunk 2]
```diff
@@ -114,14 +116,23 @@ def test_init_process(self):
 
     # confirm that the number of queries is correctly distributed across
     #  each thread in this worker process
+
```
Comment (GeorgeJahad) [inline contrib/grinder/scripts/tests.py]: use num_nodes instead of "2" in the calculations below.

Comment (goru97) [inline contrib/grinder/scripts/tests.py]: Done

File: contrib/grinder/scripts/query.py (hunk 1 above)
Comment (goru97) [inline contrib/grinder/scripts/query.py]: She's actually creating new variable annotations_per_tenant but using existing num_tenants.

Comment (tilogaat) [inline contrib/grinder/scripts/query.py]: @goru97 i am using the existing num_tenants currently, but planning to have a separate num_annotations_tenants

**Comment by GeorgeJahad** [comment]
+1

**Comment by goru97** [comment]
Merged using git! Closing it now.

**Pull request closed without merging**